### PR TITLE
C# and PowerShell versions

### PR DESCRIPTION
**Pull request by dmfabritius**
I was showing the original code to a friend and wanted to show how it could be done in other languages as well. This is not intended to be production code, it is just for demonstration/educational purposes.

### CODE REVIEW

**Comment by mikepound** [comment]
Am happy to merge this. Could you drop the user specific config file? That way when people fire it up it'll default to theirs. Specifically - the .user file.

**Comment by dmfabritius** [comment]
Sure, no problem

File: csharp/pwned-search.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+using System.IO;
+
+namespace pwned_search {
+    class Program {
+        static void Main(string[] args) {
+            Console.Write("Enter the password to check: ");
+            string plaintext = Console.ReadLine();
+
+            SHA1 sha = new SHA1CryptoServiceProvider();
+            byte[] data = sha.ComputeHash(Encoding.ASCII.GetBytes(plaintext));
```
Comment (d-chris) [inline csharp/pwned-search.cs:14]: i'm not into CS or Powershell but change to utf-8 #9

**Review by d-chris — COMMENTED** [review]
change encoding from ascii to utf-8

**Pull request merged**